ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-gas-limit/advanced-gas-fee-gas-limit.js

```diff
@@ -36,6 +38,12 @@ const AdvancedGasFeeGasLimit = () => {
     setGasLimit(value);
   };
 
+  useEffect(() => {
+    if (!initialGasValues.gasLimit) {
```

Comment:
What do you think about using `!Boolean(initialGasValues.gasLimit)` rather than just `!`, in case this is ever a number? Same goes for the other three instances of this.

Reply:
Weird but it seems to fail lint:

<img width="616" alt="Screenshot 2022-01-19 at 3 39 46 PM" src="https://user-images.githubusercontent.com/2182307/150109731-67b8b2e0-e31c-4d22-9e72-55706c38acce.png">

Reply:
Hmm, weird. Perhaps `if (initialGasValues.gasLimit === undefined || initialGasValues.gasLimit === null) {`, then?

Reply:
I removed this part of code, thus resolving it